MAINTAINERS.md

```diff
@@ -33,7 +33,7 @@
 | [react-native-geolocation](https://github.com/react-native-community/react-native-geolocation) | [@janicduplessis](https://github.com/janicduplessis), [@matt-oakes](https://github.com/matt-oakes), [@michalchudziak](https://github.com/michalchudziak) | active |
 | [react-native-google-analytics](https://github.com/react-native-community/react-native-google-analytics) | [@brentvatne](https://github.com/brentvatne) | inactive(?) |
 | [react-native-google-signin](https://github.com/react-native-community/react-native-google-signin) | [@vonovak](https://github.com/vonovak) | active |
-| [react-native-hooks](https://github.com/react-native-community/react-native-hooks) | [@dabit3](https://github.com/dabit3), [@melihberberolu](https://github.com/melihberberolu) | active |
+| [react-native-hooks](https://github.com/react-native-community/react-native-hooks) | [@dabit3](https://github.com/dabit3), [@melihberberolu](https://github.com/melihberberolu), [@pvinis](https://github.com/pvinis) | active |
```

Comment:
Do you know if the others are still active?

Reply:
As discussed in Discord I would also like to be a maintainer on this package, should I add myself in a follow up PR? ☺️

Reply:
Oh yeah, please! I'll send you an invite for the organisation.

Reply:
I can add you in this PR, and then once we know the status of the order two, we can merge.

Reply:
Sounds perfect 👍